src/subcommands/json.rs

```diff
@@ -21,27 +21,37 @@ pub struct NotAudited {
     foreign_crates: Vec<String>,
 }
 
-pub fn json(args: Vec<String>, max_age: std::time::Duration) -> Result<(), std::io::Error> {
+pub fn json(
+    args: Vec<String>,
+    diffable: bool,
+    max_age: std::time::Duration,
+) -> Result<(), std::io::Error> {
     let mut output = StructuredOutput::default();
```

Comment:
I think this should be destructured into references to its fields without `..` such that, in the case a new field is added, we don't forget to check for the diffability in the future.

Reply:
I'm afraid I don't understand what that would look like. Could you provide an example?

Reply:
```rustlet mut output = StructuredOutput::default();
let StructuredOutput {
    not_audited: NotAudited {
        ref mut local_crates,
        ref mut foreign_crates,
    },
    ref mut crates_io_crates,
} = &mut output;
```

But to be perfect honest it was a spur-of-the-moment thought and it doesn't look as pretty when written out. Since fields are assigned to in both branches it doesn't lead to what I had intended it to do, it won't warn about unused fields in either branch.